test/db.go

```diff
@@ -54,7 +54,7 @@ func createLocalDB(t *testing.T, dbName string) {
 	}
 	err := createDB.Run()
 	if err != nil && !Quiet {
-		fmt.Println("createLocalDB returned error:", err)
+		fatalError(t, "createLocalDB returned error: %s", err)
```

Comment:
Please don't. Createdb can fail if the database already exists locally which is fine.